mysql_scripts/test_data/FullDBTestData.sql

```diff
@@ -566,6 +567,36 @@ INSERT INTO Answer (possibleAnswerId, appointmentId, questionId)
 -- end answer
 
 
+
+-- virtualAppointmentConsent 
+INSERT INTO VirtualAppointmentConsent (reviewConsent, virtualConsent, signature, appointmentId)
+	VALUES (0, 1, "test_signature_one", @appointment_appointment1Id);
+INSERT INTO VirtualAppointmentConsent (reviewConsent, virtualConsent, signature, appointmentId)
+	VALUES (1, 1, "test_signature test_last", @appointment_appointment2Id);
+INSERT INTO VirtualAppointmentConsent (reviewConsent, virtualConsent, signature, appointmentId)
+	VALUES (1, 0, "Barry Test", @appointment_appointment3Id);
+INSERT INTO VirtualAppointmentConsent (reviewConsent, virtualConsent, signature, appointmentId)
+	VALUES (0, 0, "signature fo\' testing", @appointment_appointment4Id);
+INSERT INTO VirtualAppointmentConsent (reviewConsent, virtualConsent, signature, appointmentId)
+	VALUES (0, 1, "", @appointment_appointment5Id);
+-- end virtualAppointmentConsent
+
+
+
+-- selfserviceappointmentrescheduletoken
+INSERT INTO SelfServiceAppointmentRescheduleToken (token, appointmentId)
```

Comment:
Oh nice, thanks for doing this--drove me nuts that I had to sign up for an appointment every time

Reply:
haha np